ArtemShchepets/src/main/java/week1/controller/Terminal.java

```diff
@@ -249,85 +211,80 @@ public void createBill() {
                     actualSizeOfBills++;
                     currentBillIndex = actualSizeOfBills - 1;
 
+                    return true;
                 } else {
 
                     bills[actualSizeOfBills] = newBill;
 
                     actualSizeOfBills++;
                     currentBillIndex = actualSizeOfBills - 1;
 
+                    return true;
                 }
             }
-
-            System.out.println("***BILL IS CREATED***");
         }
     }
 
-    public void addProductToBill() {
+    public boolean closeAllPreviousBills(Time closeTime) {
 
```

Comment:
You should not have closeTime parameters. System should automaticaly get time. I think better when you will call smth Utils.getCurrentTime